src/components/AddProject.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+<div id="add-project-container">
+    <div class="field">
+        <label for="project-title">TITLE</label>
+        <input type="text" id="project-title" v-model="title">
+    </div>
+    <div class="field">
+        <label for="project-details">DETAILS</label>
+        <textarea name="" id="project-details" cols="30" rows="10" v-model="detail"></textarea>
+    </div>
+    <router-link to="/" class="nav-menu "><button @click="addProject" class="btn" v-if="!this.id" :disabled="title.length === 0 || detail.length === 0">Add Project</button></router-link>
```

Comment:
@anubha-innotech If you are using a router link, do not use the button with them, especially the click function on them.